extensions/ccui/uiwidgets/scroll-widget/UIScrollView.js

```diff
@@ -344,7 +346,7 @@ ccui.ScrollView = ccui.Layout.extend(/** @lends ccui.ScrollView# */{
     addChild: function (widget, zOrder, tag) {
         if(!widget)
             return false;
-        if(this._clippingEnabled && this._isInContainer(widget) === false)
+        if(this._isInContainer(widget) === false)
```

Comment:
We won't need to check `this._clippingEnabled` here because `_isInContainer` will do the check and return true